tests/hazmat/primitives/test_rsa.py

```diff
@@ -1985,10 +1985,11 @@ def test_recover_prime_factors(self, vector):
             private["private_exponent"]
         )
         # Unfortunately there is no convention on which prime should be p
-        # and which one q. The function we use always makes p < q, but the
+        # and which one q. The function we use always makes p > q, but the
         # NIST vectors are not so consistent. Accordingly, we verify we've
         # recovered the proper (p, q) by sorting them and asserting on that.
         assert sorted([p, q]) == sorted([private["p"], private["q"]])
+        assert p > q, (p, q)
```

Comment:
It's set as the message for the `AssertionError` if such an error is raised, so that you can see the values that led to the assertion trigger. I guess pytest may already rewrite assertion statements so that it does something friendly, however this is very useful outside pytest contexts.

Do you want me to remove it?